Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.h

```diff
@@ -220,6 +220,13 @@ typedef void (^BUYDataGiftCardBlock)(BUYGiftCard * _Nullable giftCard, NSError *
  */
 - (NSOperation *)removeGiftCard:(BUYGiftCard *)giftCard fromCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
 
+/**
+ *	Removes the discount from the checkout.
+ *
+ *  @param checkout An existing BUYCheckout on Shopify
+ */
+- (void)removeDiscountFromCheckout:(BUYCheckout *)checkout;
```

Comment:
If we put this on `BUYCheckout` (as simply `-removeDiscount` instead of on `BUYClient`, would that be reasonable even if not on Android? I'm pretty sure we have a lot of API differences, but I understand we want `BUYClient` to be basically identical.

I actually was not expecting this to be on `BUYClient`. It's up to the app to decide when to push updates to Shopify.